src/main/migrations/20211117082401-create-difficulty-level.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const difficultyLevels = [
+  {
+    id: 1,
+    name: 'easy',
+    createdAt: new Date(),
```

Comment:
Generally good practice to have timestamp metadata for all entries for database management. Not super important here since these are enums and aren't likely to change anytime soon, but thought I'd add them anyways considering potential future scalability & maintenance!